.github/actions/common_setup/action.yml

```diff
@@ -0,0 +1,26 @@
+name: Common setup
+description: Setup necessary environments and clean the runner's temp path on ending
```

Comment:
no clean

Reply:
Thanks, it was a single common step with post-run. But it didn't work out